coalib/bearlib/aspects/decorators.py

```diff
@@ -49,3 +49,51 @@ def _new_func(self, *args, **kwargs):
         return _new_func
 
     return _func_decorator
+
+
+def map_ambiguous_setting_to_aspect(**aspectable_setting):
+    """
+    Map those bear settings to aspects in which there is some ambiguity.
+    The ambiguity is because of differing values (maybe either name or even
+    data-types) in bear settings and their corresponding aspect tastes.
+
+    This decorator can be used by ``Bear.run()`` to automatically map and
+    override value of bear settings with their equivalent taste.
```

Comment:
This decorator is used by Bear.run() to override the value in the bear setting with an equivalent taste as provided in the parameter.